dask_jobqueue/pbs.py

```diff
@@ -55,7 +55,9 @@ class PBSCluster(JobQueueCluster):
     cancel_command = 'qdel'
     scheduler_name = 'pbs'
```

Comment:
You didn't removed scheduler_name here.

Reply:
This should be fixed in newest commits on the PR.